assets/css/report-card.css

```diff
@@ -78,9 +79,32 @@
 .report-card__picture-container {
   display: flex;
   gap: 0.75rem;
-  overflow: scroll;
+  overflow-x: auto;
+  scroll-behavior: smooth;
+  padding-bottom: 0.75rem;
 }
 
+/* Scroller */
+.report-card__picture-container::-webkit-scrollbar {
+  display: block;
+  width: 6px;
+  height: 6px;
+}
+
+.report-card__picture-container::-webkit-scrollbar-track {
+  background: #f1f1f1;
```

Comment:
if this color is not inside the variables, please create a new variable https://github.com/whamester/WMDD-4885-Project/blob/whamester/fix-css-variables-reference/assets/css/_variables.css

Reply:
Okay, sure